gnuboardProject/backend/src/users/users.controller.ts

```
@@ -1,8 +1,11 @@
-import { Controller, Get, Put, Body, UseGuards, Request, Delete, Param, ForbiddenException } from '@nestjs/common';
+import { Controller, Get, Put, Body, UseGuards, Request, Delete, Param, ForbiddenException, UseInterceptors, UploadedFile } from '@nestjs/common';
 import { UsersService } from './users.service';
 import { JwtAuthGuard } from '../auth/jwt-auth.guard';
 import { UpdateUserDto } from './dto/update-user.dto';
 import { ChangePasswordDto } from './dto/change-password.dto';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { diskStorage } from 'multer';
+import * as path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)